### ISSUE TITLE
Problem ObjectUnsubscribedError

### ISSUE THREAD

**Issue by Saloukai** [wangdicoder/angular-material-dashboard#15]
Hello :)

I put a redirect button in /dashboard to return to the login page.

I have a problem in this scenario :
`From my login page, I go to /dashboard, go back to the login page, and go back to / dashboard.`

The whole page is not loaded, and an error is generated : ObjectUnsubscribedError from RootComponent.

Screen of this issue: https://ibb.co/drqoUf

Do you have any idea ?

Thanks you, have a good day !

**Comment by DenisLord** [comment]
Yeah I have the same problem. I just deleted every unsubscribe and whatever was related to those variables in html and components and added a rgba color instead of random images.
